core/templates/components/summary-tile/exploration-summary-tile.component.html

```diff
@@ -11,7 +11,7 @@
   <mat-card [ngClass]="{'oppia-activity-summary-tile-mobile': isCollectionPreviewTile, 'oppia-activity-summary-tile': !isCollectionPreviewTile, 'small-width': !isWindowLarge, 'oppia-activity-playlist-tile': isPlaylistTile}"
           *ngIf="!isRefresherExploration" [style.padding]="0" class="oppia-exploration-dashboard-card protractor-test-exploration-dashboard-card">
     <a [href]="getExplorationLink()" target="{{ openInNewWindow ? '_blank' : '_self' }}">
-      <div class="title-section protractor-test-exp-hover" [ngStyle]="{'background-color': thumbnailBgColor}">
+      <div class="title-section protractor-test-exp-hover" (mouseenter)="setHoverState(true)" (mouseleave)="setHoverState(false)" [ngStyle]="{'background-color': thumbnailBgColor}">
```

Comment:
Is this solution conceptually correct? The div on line 27 has an ngIf control on which the hover state is applied, so: (a) shouldn't that control apply here too, (b) shouldn't we only have the mouseenter in a single location?

Reply:
Moved `protractor-test-exp-hover` to line 27.